charts/flyte-core/templates/datacatalog/service.yaml

```diff
@@ -12,15 +12,17 @@ spec:
   {{- with .Values.datacatalog.service.type}}
   type: {{ . }}
   {{- end }}
+  {{- with .Values.datacatalog.service.clusterIP}}
+  clusterIP: {{ . }}
+  {{- end }}
   ports:
-  - name: http
```

Comment:
This was broken. Based on the config it actually listens on 8080, not 8088: https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/values.yaml#L953